tests/fixtures/apples

```diff
@@ -1 +0,0 @@
-/Users/stefan/src/broccoli-stew/tmp/funnel-dest_O6Pex7.tmp
```

Comment:
it should have affected everyone, giving specific usage patterns.

Anyways, im glad its gone now